assets/src/app/ato/Products/Static.tsx

```diff
@@ -2,9 +2,12 @@ export const ProductIdOverridesMap: {[Identifier: string]: string} = {
     coreos4: 'coreos-4',
     idm: 'identity-management',
     ocp3: 'openshift-container-platform-3',
+    ocp4: 'openshift-container-platform-4',
     osp13: 'openstack-platform-13',
+    rhel7: 'enterprise-linux-7',
+    rhel8: 'enterprise-linux-8',
```

Comment:
Why do you see a need adding those items?

This whole map is here only to maintain links that people may have bookmarked 10+ months ago. CommplianceAsCode/redhat project used to have different product names in both branches. After migration of ComplianceAsCode/redhat to github actions, I consolidated old names. This map is here only to keep the old names working. No need to add new ones.